src/ddmd/lexer.d

```diff
@@ -75,7 +75,7 @@ bool issinglechar(char c)
     return (cmtable[c] & CMsinglechar) != 0;
 }
 
-static this()
+shared static this()
```

Comment:
I am just looking at this from my phone, but isn't `cmtable` `immutable ubyte[256]`?

Reply:
Yes. I also wondered why it's not forbidden to change it in a thread local ctor.